crates/bevy_render/src/extract_param.rs

```diff
@@ -12,7 +12,7 @@ use std::ops::{Deref, DerefMut};
 /// contained in [`MainWorld`]. This parameter only works for systems run
 /// during the [`ExtractSchedule`](crate::ExtractSchedule).
 ///
-/// This requires that the contained [`SystemParam`] does not mutate the world, as it
+/// This requires that the contained [`SystemParam`] not mutate the world, as it
```

Comment:
Looks like you accidentally removed a word here

Reply:
That was intentional — the sentence without "does" sounds more idiomatic and correct to my ears.  I can add the word back if desired.

Reply:
I'm not a native speaker, but it's a bit awkward without 'does' IMO.
I think it's correct grammar but 'do' isn't usually omitted in modern English as far as I know.

Reply:
It's a subtle corner case, in all fairness.  And there's a possibility that I'm wrong, even though I am a native speaker.  But that feeling you got with my change was the same one I got when reading it with "does" in it, before the change.

Reply:
This should just be rephrased to avoid this. E.g. "As a result, the contained `SystemParam` must not mutate the world..."